contentcuration/contentcuration/frontend/shared/vuex/syncProgressPlugin/index.js

```diff
@@ -0,0 +1,21 @@
+import syncProgressModule from './syncProgressModule';
+import db from 'shared/data/db';
+import { CHANGES_TABLE } from 'shared/data/constants';
+
+const SyncProgressPlugin = store => {
+  store.registerModule('syncProgress', syncProgressModule);
+
+  db.on('changes', function(changes) {
+    const changesTableUpdated = changes.some(change => change.table === CHANGES_TABLE);
+    if (!changesTableUpdated) {
+      return;
+    }
+
+    db[CHANGES_TABLE].toCollection()
+      .filter(c => !c.errors)
+      .count()
```

Comment:
Can put `.limit(1)` on the line before as we are just doing an exists check, so we don't need to query all the elements.

https://dexie.org/docs/Collection/Collection.limit()